lamp/lib/authentication.php

```diff
@@ -449,7 +449,13 @@ function does_user_have_access($uid, $channel_id) {
     function is_user_instructor($course_id)
     {
         include_once("lib/course-handler.php");
-        $uid = $_SESSION['uid'];
+        include_once("lib/page-setup.php");
```

Comment:
Why are we including page set up here? 
authentication.php is included in page-setup.php, so this is unnecessary.